ui/server.py

```diff
@@ -328,4 +328,5 @@ def update_discovery_group():
         return 'OK', 200
 
 
-app.run(host='0.0.0.0', port=5000)
+if __name__ == "__main__":
```

Comment:
single quotes for strings in python (in fact you have single quotes at the following line). Double quotes are used for docstrings